local/README.md

```diff
@@ -0,0 +1,44 @@
+### Local deployment files for swift-ui stack
+This folder contains the docker-compose files for deploying a complete local
+instance of the swift-ui, with sharing and access request support.
+
+When deployed, the service will be available on port 8080.
+
+The deployment uses the file `envs.txt` as a source for the environment
```

Comment:
The dockerfiles actually expect a `envs.sh` file.

Reply:
Fixed